### ISSUE TITLE
[GraphOptz] Fuse BatchNorm into FullyConnected

### ISSUE THREAD

**Issue by mciprian13** [pytorch/glow#5339]
We currently have an optimization to fuse BatchNorm into Conv2D but we need also for FullyConnected (it is similar).
